City-of-Helsinki/drupal-helfi-etusivu
# UHF-5666: Created target group paragraphs and added dummy templates

# Target group links [UHF-5666](https://helsinkisolutionoffice.atlassian.net/browse/UHF-5666)
Styles target group links "Juuri sinulle" to etusivu-instance.

## What was done
* Added a new paragraph type to landing pages
* Set the char limits and required fields as instructed
* Styled the paragraph type and its subtype
* Added stylelint to this projects subtheme

![image](https://user-images.githubusercontent.com/1191667/170227953-cb680e05-3676-45ca-9e95-802b7f46a3e2.png)
![image](https://user-images.githubusercontent.com/1191667/170228020-cb9edf53-44ff-46d6-827b-679ea54874ed.png)


## How to install
* Make sure your instance is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh`
* Update the HDBT theme
    * `composer require [THIS BRANCH]` (I do not know how to do this with composer)
    * `composer require drupal/hdbt:dev-UHF-5666-for-you-paragraph`
* Run `make drush-cr`

## How to test
* Get this branch, get the hdbt branch
* Add a new landing page, add the `Target group links` component
* Add an item with internal link
* Add an item with external link
* Check that external link icon does not fall on its own row alone
* Check xs, m, l breakpoints, they all have some minor differences

## Other PRs
* https://github.com/City-of-Helsinki/drupal-hdbt/pull/325
